DOC: EHT Case Study Suggestions

rossbar opened the pull request:
A few suggestions for the EHT-imaging case study based on my comments in #175 .

I tried to make the commits relatively fine-grained so that individual changes can be reverted as you see fit. The main themes of these changes are:
 1. Try to communicate some of the concepts with more precise language
 2. Try to tighten the focus on NumPy and the role played in generating the black hole image.

I had one additional thought on the last graphic, i.e. the one captioned "Key NumPy Capabilities Used". To me, the caption is a bit out of sync with the image itself, which focuses a bit more on some specific technical challenges faced by the EHT team. I think the figure could have a more numpy-centric theme. For example, you could consider the following categories/descriptions:
 
**Flexibility** -> NumPy's generic `ndarray` structure is suitable for representing numerical data of many types and dimensions - from 1D time-series data to 2D images and beyond!
**Simplicity** -> NumPy's array syntax allows users to express complex numerical manipulations clearly and concisely
**Foundational** -> NumPy is the fundamental package for scientific computing in Python, supporting and interfacing with the incredibly powerful scientific Python ecosystem.

These are just a couple suggestions - the wording would likely needs some work and all that, but I do think having a more NumPy-centric focus for the final figure would be an improvement.

rgommers commented on content/en/case-studies/blackhole-image.md:
```diff
@@ -60,75 +64,71 @@ enough to read a newspaper in New York from a sidewalk café in Paris!
+of the final image of the black hole.
 
+[astropy]: https://www.astropy.org/
+[mpl]: https://matplotlib.org/
```
maybe make these normal (`[]()` markdown links?

rgommers replied:
and the `ehtim` one above as well?

rgommers replied:
oh, they were normal links before - I think that'd be more robust and just as easy

rossbar replied:
Dealer's choice - I often use the markdown [name][key] link style as it makes the source text easier to read (especially when the links themselves are very long). I don't mind either way though.

rossbar replied:
> I think that'd be more robust and just as easy

One last thought: both syntaxes are [commonmark compliant](https://commonmark.org/help/), so robustness shouldn't be an issue - more of a personal preference

rgommers replied:
okay never mind then. I have actually never seen this used, but if it's compliant should be okay

shaloo replied:
I'll try to take a look at these today. Thanks @rossbar and @rgommers.

rossbar commented:
Thanks for taking a look at this! A few of my comments below:

> IMHO, Key Goals and results would read better if each bullet has a 'bold text' heading. For e.g. for the last bullet:

Sounds good to me :+1: 

> Regarding the new bullet point "The EHT is an exciting new tool..." What is the goal and what is the result in this case? I'm not sure I understand from the text. The last one looks ok.

I hear what you're saying - the bullet doesn't neatly fit into the "goal and results" category. I suppose "the goal" in this case is to provide a "new" instrument that is sensitive to a regime of objects that astronomers have never before been able to directly investigate. In that sense I tried to stick with what I interpreted as the theme from the original point, but perhaps I lost the plot a bit. Please let me know how it could be improved.

> NumPy's Role - First para - some context is needed - see other detailed comment.

You make some great points here - since there is a lot to cover, I will try to address them separately:

>  Since NumPy was involved in imaging tasks for a layman it is important to understand it in simple terms, beyond data analysis and technicalities, why imaging IS important in BH context.

I agree that the paragraph that I added to replace the original is much more technical, and perhaps too much so. My goal was to provide a bit of background for *imaging* generally and the approach that the EHT team used for the imaging analysis. I think this is a strong point because it highlights (as I mention in the text) the importance of reproducibility and collaboration, which are two pillars of science that the scientific Python ecosystem does a great job of supporting.

If the paragraph that I added is too technical and you decide to go with the original paragraph (which does give a nice overview) I'd have a couple additional suggestions:

> Imaging is crucial as it can help to predict not only the black hole mass but also rule out whether a black hole could be a wormhole

This statement probably needs a citation. I don't have domain-specific expertise here, but in the research I've done on the EHT I haven't seen any mention of using imaging to discriminate against other high-mass objects. This very well may be true, but I think it is a specific enough statement to require a reference.

> As Katie Bouman mentions in her TED talk

I hadn't considered this from the "honeypot"/SEO angle that you bring up. Also I ended up removing the link not because I don't think the video is good, but because it didn't fit in the imaging narrative that I was constructing.

> ‘It is like taking a picture of an orange on the surface of the moon.’

This is essentially a re-statement of the resolution analogy (i.e. reading a newspaper in NY from Paris) and is more related to the physical parameters of the instrument itself rather than the imaging methodology. Again, maybe these concerns are too technical.

shaloo commented:
> I hear what you're saying - the bullet doesn't neatly fit into the "goal and results" category. I suppose "the goal" in this case is to provide a "new" instrument that is sensitive to a regime of objects that astronomers have never before been able to directly investigate. In that sense I tried to stick with what I interpreted as the theme from the original point, but perhaps I lost the plot a bit. Please let me know how it could be improved.
> 

> If the paragraph that I added is too technical and you decide to go with the original paragraph (which does give a nice overview) I'd have a couple additional suggestions:
> 
> > Imaging is crucial as it can help to predict not only the black hole mass but also rule out whether a black hole could be a wormhole
> 

> This statement probably needs a citation. I don't have domain-specific expertise here, but in the research I've done on the EHT I haven't seen any mention of using imaging to discriminate against other high-mass objects. This very well may be true, but I think it is a specific enough statement to require a reference.
> 
> > As Katie Bouman mentions in her TED talk
> 
> I hadn't considered this from the "honeypot"/SEO angle that you bring up. Also I ended up removing the link not because I don't think the video is good, but because it didn't fit in the imaging narrative that I was constructing.
> 
> > ‘It is like taking a picture of an orange on the surface of the moon.’
> 
> This is essentially a re-statement of the resolution analogy (i.e. reading a newspaper in NY from Paris) and is more related to the physical parameters of the instrument itself rather than the imaging methodology. Again, maybe these concerns are too technical.

About Citation: 
There were several articles that suggested this and I think Katie Bowman too.  I'll have to re-har her TED talk to identify the exact time where she said this.  One of the articles was https://www.forbes.com/sites/startswithabang/2019/04/11/10-deep-lessons-from-our-first-image-of-a-black-holes-event-horizon/#39c9f5655e64.  I'll try to get you the source over the weekend.

I think, it is really good to provide technical specifics but it is equally important for new users to understand the implications in simple terms.  And not everyone knows the technicalities at the start so analogies help them create a mind map and then delve deeper into specifics if they choose to.

rossbar commented:
Re: the citation:

On the one hand, it's great to have references to highly-trafficked articles etc. but I'd be hesitant to include such "pop" science articles as references as they are not primary sources.

I did a quick scan through [the six original papers (links at the end of the press release)](https://eventhorizontelescope.org/press-release-april-10-2019-astronomers-capture-first-image-black-hole) for mentions of wormholes. There is a a mention in [the fifth paper](https://iopscience.iop.org/article/10.3847/2041-8213/ab0f43) - the wording there however doesn't appear to completely eliminate wormholes as a possibility, but to narrow-down the set of conditions where a wormhole could generate a shadow like the one observed. 

Either way, I think it would be best to link to the EHT team's own work.

shaloo commented:
> Thanks for taking a look at this! A few of my comments below:
> 
> > IMHO, Key Goals and results would read better if each bullet has a 'bold text' heading. For e.g. for the last bullet:
> 
> Sounds good to me 👍
> 
> > Regarding the new bullet point "The EHT is an exciting new tool..." What is the goal and what is the result in this case? I'm not sure I understand from the text. The last one looks ok.
> 
> I hear what you're saying - the bullet doesn't neatly fit into the "goal and results" category. I suppose "the goal" in this case is to provide a "new" instrument that is sensitive to a regime of objects that astronomers have never before been able to directly investigate. In that sense I tried to stick with what I interpreted as the theme from the original point, but perhaps I lost the plot a bit. Please let me know how it could be improved.
> 
> > NumPy's Role - First para - some context is needed - see other detailed comment.
> 
> You make some great points here - since there is a lot to cover, I will try to address them separately:
> 
> > Since NumPy was involved in imaging tasks for a layman it is important to understand it in simple terms, beyond data analysis and technicalities, why imaging IS important in BH context.
> 
> I agree that the paragraph that I added to replace the original is much more technical, and perhaps too much so. My goal was to provide a bit of background for _imaging_ generally and the approach that the EHT team used for the imaging analysis. I think this is a strong point because it highlights (as I mention in the text) the importance of reproducibility and collaboration, which are two pillars of science that the scientific Python ecosystem does a great job of supporting.
> 
> If the paragraph that I added is too technical and you decide to go with the original paragraph (which does give a nice overview) I'd have a couple additional suggestions:
> 
> > Imaging is crucial as it can help to predict not only the black hole mass but also rule out whether a black hole could be a wormhole
> 
> This statement probably needs a citation. I don't have domain-specific expertise here, but in the research I've done on the EHT I haven't seen any mention of using imaging to discriminate against other high-mass objects. This very well may be true, but I think it is a specific enough statement to require a reference.
> 
> > As Katie Bouman mentions in her TED talk
> 
> I hadn't considered this from the "honeypot"/SEO angle that you bring up. Also I ended up removing the link not because I don't think the video is good, but because it didn't fit in the imaging narrative that I was constructing.
> 
> > ‘It is like taking a picture of an orange on the surface of the moon.’
> 
> This is essentially a re-statement of the resolution analogy (i.e. reading a newspaper in NY from Paris) and is more related to the physical parameters of the instrument itself rather than the imaging methodology. Again, maybe these concerns are too technical.

@rossbar, do you plan to get this to a closure?  Let me know if you are waiting on me for something here.  Afaik, we are good to go other than a couple of edits that you were planning to make.  You can remove the citation request above as per Ralf's note.  Thank you for your valuable inputs.

rossbar commented:
Thanks for the ping - I was under the impression that there wasn't anything left for me to do: it seems to me the big thing is a decision needs to be made regarding the "NumPy's role" paragraph, as my suggestion and your comments significantly diverge. As I mention [here](https://github.com/numpy/numpy.org/pull/196#issuecomment-603368437), sticking with the original paragraph could work, but if so there are some additional things that should be followed up. I leave the decision up to you!

> You can remove the citation request above as per Ralf's note.

I don't understand this statement - if you could provide a little more specific info about what is meant here I'd be happy to follow up.

shaloo commented:
Wrt citation, I meant we will go by what Ralf said above to focus on technical specifics rather than pop culture. So I guess we are in sync now. I will update the paragraph as per your suggestion. Thanks for your inputs.

rossbar commented:
Unless I'm missing something (maybe there was a conversation on a different medium like slack?) the suggestion to focus on technical aspects over the pop-sci take was actually my own - as such, I leave the overall decision about which direction to go to you!

Aside from that major point though, the only other action items from the last review that I noticed was your suggestion of adding a bolded heading to each bullet point in the "Key Results and Objectives" section. I took a stab at this in a97973e - let me know what you think.

Please let me know if there are outstanding issues that I missed, or if you have new suggestions upon re-review.

rgommers commented:
@shaloo are you now agreeing with these changes?

shaloo commented:
Yes, @rgommers . Thx.

rgommers commented:
Merged, thanks @rossbar, and thanks for reviewing @shaloo!
